scripts/build_provenance

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+readonly BUILD_SCRIPT=$1
+readonly OUTPUT_PATH=$2
+
+readonly BUILD_COMMAND="./scripts/docker_run ${BUILD_SCRIPT}"
+eval "${BUILD_COMMAND}"
+readonly BINARY_SHA_256_HASH=$(sha256sum "${OUTPUT_PATH}" | cut -d " " -f 1)
+
+readonly COMMIT_HASH=$(git rev-parse --verify HEAD)
+
+mkdir -p "./provenance"
+mkdir -p "./provenance/${BINARY_SHA_256_HASH}"
+
+readonly PROVENANCE=$(cat << EOF >> "./provenance/${BINARY_SHA_256_HASH}/${COMMIT_HASH}.toml"
```

Comment:
Why is this being read into PROVENANCE?

Reply:
Good point, let's jettison that variable and write directly to the the file.